### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by ynojima**
***
<a href="https://pr-preview.s3.amazonaws.com/ynojima/webauthn/pull/736.html" title="Last updated on Jan 6, 2018, 5:50 AM GMT (8311821)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/736/0ed6257...ynojima:8311821.html" title="Last updated on Jan 6, 2018, 5:50 AM GMT (8311821)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1600,7 +1600,7 @@ associated.
     ::  A [=URL serializer|serialized=] URL which resolves to an image associated with the entity. For example, this could be
         a user's avatar or a [=[RP]=]'s logo. This URL MUST be an [=a priori authenticated URL=]. [=Authenticators=] MUST
         accept and store a 128 byte minimum length for a icon members's value.
-        Authenticators MAY ignore a icon members's value if its length is greater than 128 byes.
+        Authenticators MAY ignore a icon members's value if its length is greater than 128 bytes.
```
Comment (emlun) [inline index.bs]: Might as well fix "a icon" to "an icon" too while we're at it.

**Comment by emlun** [comment]
I'll do it in #687.

**Comment by emlun** [comment]
These changes are now merged into master via f716b7fcfa4be4566a88a6f7b18b51804e66c721 in #687, I propose closing this.

**Comment by equalsJeffH** [comment]
closing this PR per https://github.com/w3c/webauthn/pull/736#issuecomment-358403456, thanks @ynojima !!

**Comment by equalsJeffH** [comment]
thx also @emlun for incorp'g these (editorial) changes in another PR :)

**Pull request closed without merging**